README.md

```diff
@@ -46,7 +46,7 @@ You’d like to contribute? Don’t hesitate to check out our [contributing guid
 
 ## 📖 Documentation
 
-See our [Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
+See the [Documentation of this SDK](https://meilisearch.github.io/meilisearch-python/), our [Meilisearch Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
```

Comment:
I’m a bit confused about this—it seems like the link given (https://meilisearch.github.io/meilisearch-python/) is a deploy of this README. So isn't this kind of like linking to itself?

Reply:
It should deploy the documentation after the release the build is triggered by pushing new tag

Reply:
@dichotommy, I trigger the action the link should work now!